### ISSUE TITLE
YouTubeRebornPlus v19.10.5-4.2.6

### ISSUE THREAD

**Issue by superunknowndude** [arichornlover/YouTubeRebornPlus#107]
https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242

You're welcome. :)

**Comment by superunknowndude** [comment]
> I had to remove the IPAs to comply to a DMCA from Google.

I know ... It's a bummer. :(

**Comment by rainerlogan** [comment]
> https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242
> 
> You're welcome. :)

I made this account just to say GOD BLESS YOU. THANK YOU SO MUCH. I have no idea what I'm doing here, and I tried my best at building the ipa by following all the instructions and I almost punched a hole in my laptop because I kept getting errors with like "exited with error code 2" and "sysroot not found" whatever that means. Maybe because I'm using a windows laptop and have ABSOLUTELY NO IDEA WHAT I'M DOING and all this code stuff + GitHub's UI was making me go crosseyed from staring at things I don't understand for so long. Literally like my eyes were drooping from stress and exhaustion. God bless you. You're a hero

**Comment by superunknowndude** [comment]
> > https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242
> > You're welcome. :)
> 
> I made this account just to say GOD BLESS YOU. THANK YOU SO MUCH. I have no idea what I'm doing here, and I tried my best at building the ipa by following all the instructions and I almost punched a hole in my laptop because I kept getting errors with like "exited with error code 2" and "sysroot not found" whatever that means. Maybe because I'm using a windows laptop and have ABSOLUTELY NO IDEA WHAT I'M DOING and all this code stuff + GitHub's UI was making me go crosseyed from staring at things I don't understand for so long. Literally like my eyes were drooping from stress and exhaustion. God bless you. You're a hero

Thank you.
All credit to @arichornlover.   They're the real hero. :)

**Comment by KXRIST** [comment]
sorry about the question, answer must exist somewhere, can find it. how to create ios ipa file from zip ?

**Comment by rainerlogan** [comment]
> > > https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242
> > > You're welcome. :)
> > 
> > 
> > I made this account just to say GOD BLESS YOU. THANK YOU SO MUCH. I have no idea what I'm doing here, and I tried my best at building the ipa by following all the instructions and I almost punched a hole in my laptop because I kept getting errors with like "exited with error code 2" and "sysroot not found" whatever that means. Maybe because I'm using a windows laptop and have ABSOLUTELY NO IDEA WHAT I'M DOING and all this code stuff + GitHub's UI was making me go crosseyed from staring at things I don't understand for so long. Literally like my eyes were drooping from stress and exhaustion. God bless you. You're a hero
> 
> I did fix the error. I didn’t even realize it had an error. But the Artifact is easier way of obtaining the file!

Thank you so much! I figured I was just making a mistake myself the entire time because I had no idea what I was doing and my brain was going so slow. I just kept getting strange errors and couldn’t find someone with the exact issue online, but I tried to follow the instructions on your wiki? page but I must have made an error along the way.

Thank you for your dedication to us and for not throwing in the towel when Google pushed that cruel DMCA stuff on you. We appreciate you so much!

**Comment by rainerlogan** [comment]
> sorry about the question, answer must exist somewhere, can find it. how to create ios ipa file from zip ?

If you can’t figure it out, I think user “superunknowndude” in this same thread sent a artifact download link of the ipa that’s already decrypted and built, so all you have to do is download it, unzip the ipa out of the zip file, and then sync it to your phone with whatever sideloading program you use. 

I’m not sure if I’m allowed to say this or to repost the link, so I apologize profusely if I accidentally crossed a line. Here’s the link. Clicking on it should immediately begin the download of the zip file with the app you’re looking for inside. All credit goes to the wonderful creator of this and to the godsend who posted this link in the comments as well. :)

https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242

**Comment by KXRIST** [comment]
thanks for the ipa, it works, although i want to have the option to create it on my own after unziping the files. Is there any guide from

**Comment by egrandblvd** [comment]
Can someone please explain how to create ipa’s? Is a way to send ipa’s via emails? 😫 Help!

**Comment by aeskellion** [comment]
> https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242
> 
> You're welcome. :)

Maybe I'm stoopid but when I click on this link I get a 404 error and no download. Has it been taken down too ?

**Comment by egrandblvd** [comment]
Yeah! Thanks to Google we’re not getting ipa’s anymore!Sent from my iPhoneOn Mar 28, 2024, at 3:42 AM, aeskellion ***@***.***> wrote:﻿

https://github.com/superunknowndude/YouTubeRebornPlus/actions/runs/8320524283/artifacts/1333770242
You're welcome. :)

Maybe I'm stoopid but when I click on this link I get a 404 error and no download. Has it been taken down too ?

**Comment by superunknowndude** [comment]
@aeskellion @egrandblvd @KXRIST 
I compile it once a week when the app shuts down on me because of the 7-day dev limit thing, if there is an update available. Otherwise I just reload the last compiled version for another week.  I've been away the last 2 weeks, unfortunately, so I haven't been able to post an update. But, I'm back now. :)

I just updated the link under a new issue. If you still need the ipa, I've added it to dropbox in case the artifact link 404's.

**Comment by egrandblvd** [comment]
Can you provide a link to the drop box? Thanks!😊 Sent from my iPhoneOn Apr 6, 2024, at 12:23 PM, superunknowndude ***@***.***> wrote:﻿
@aeskellion @egrandblvd @KXRIST
I compile it once a week when the app shuts down on me because of the 7-day dev limit thing, if there is an update available. Otherwise I just reload the last compiled version for another week.  I've been away the last 2 weeks, unfortunately, so I haven't been able to post an update. But, I'm back now. :)
I just updated the link under a new issue. If you still need the ipa, I've added it to dropbox in case the artifact link 404's.

**Comment by egrandblvd** [comment]
I found it! Thanks!🙏 Sent from my iPhoneOn Apr 6, 2024, at 3:17 PM, ryan carey dixon ***@***.***> wrote:﻿Can you provide a link to the drop box? Thanks!😊 Sent from my iPhoneOn Apr 6, 2024, at 12:23 PM, superunknowndude ***@***.***> wrote:﻿
@aeskellion @egrandblvd @KXRIST
I compile it once a week when the app shuts down on me because of the 7-day dev limit thing, if there is an update available. Otherwise I just reload the last compiled version for another week.  I've been away the last 2 weeks, unfortunately, so I haven't been able to post an update. But, I'm back now. :)
I just updated the link under a new issue. If you still need the ipa, I've added it to dropbox in case the artifact link 404's.

**Comment by rainerlogan** [comment]
> @aeskellion @egrandblvd @KXRIST I compile it once a week when the app shuts down on me because of the 7-day dev limit thing, if there is an update available. Otherwise I just reload the last compiled version for another week. I've been away the last 2 weeks, unfortunately, so I haven't been able to post an update. But, I'm back now. :)
> 
> I just updated the link under a new issue. If you still need the ipa, I've added it to dropbox in case the artifact link 404's.

As someone who still couldn’t figure out how to build the ipa myself even when I downloaded a correct decrypted YT ipa, can I just say thank you so much again? The amount of brainpower and time you saved me with this cannot be put lightly, I was literally almost pulling my hair out because I still don’t know coding well enough to even comprehend the reason for the errors I got, and google/youtube tutorials didn’t seem to help at all because they were for similar errors but not the exact error I had. Maybe it’s because I tried to build the ipa using a Windows Laptop? But I then tried it on iPhone’s Safari browser and that didn’t work either when I tried, I still kept getting the Error 2 thing I mentioned in my older comment.

Either way, thank you BOTH (you and arichornlover) so much for your dedication to upload this even with the DMCAs and essentially putting yourself at risk to save us some time. I’m sure you did the OP a ton of favors too since they can no longer host the ipa themselves and they probably have been dealing with an influx of ppl like me who have nothing but confused questions and no real answers 😫 

Is the issue with iSponsorBlock causing the app to consistently crash when trying to play a video fixed? I managed to get it working by disabled iSponsorBlock entirely, I tried using the version spoofer thing someone mentioned in the Issues section but to no avail. If you don’t know why this error persists, that’s totally fine too. You’ve already helped us an incredible amount :)
